V/VMEC/build_tarballs.jl

```diff
@@ -65,11 +65,10 @@ products = [
 # Dependencies that must be installed before this package can be built
 dependencies = [
     # MbedTLS is an indirect dependency, fix the version for building 
-    BuildDependency(PackageSpec(name = "MbedTLS_jll", version = v"2.24.0")),
-    Dependency("MPICH_jll"; platforms=filter(!Sys.iswindows, platforms)),
-    Dependency("OpenBLAS_jll"),
+    BuildDependency(PackageSpec(name = "MbedTLS_jll")),
+    Dependency("MPItrampoline_jll"; platforms=filter(!Sys.iswindows, platforms)),
```

Comment:
For MPITrampoline you probably want to follow the example of the other packages using it.

Reply:
I followed the examples of the other packages using MPItrampoline and it seems to build correctly.  If you approve, could this get merged soon?  This is intended to resolve MPI difficulties that have been occurring on HPC systems.